services/blockchain-indexer/shared/indexer/blockchainIndex.js

```diff
@@ -174,14 +174,18 @@ const indexBlock = async job => {
 						block.generatorAddress, blockReward, commissionAmount,
 					);
 
+					logger.trace(`Increasing commission for validator ${block.generatorAddress} by ${commissionAmount}.`);
 					await validatorsTable.increment({
 						increment: { totalCommission: BigInt(commissionAmount) },
 						where: { address: block.generatorAddress },
 					}, dbTrx);
+					logger.debug(`Increased commission for validator ${block.generatorAddress} by ${commissionAmount}.`);
+					logger.trace(`Increasing self-stake rewards for validator ${block.generatorAddress} by ${commissionAmount}.`);
 					await validatorsTable.increment({
 						increment: { totalSelfStakeRewards: BigInt(selfStakeReward) },
 						where: { address: block.generatorAddress },
 					}, dbTrx);
+					logger.debug(`Increased self-stake rewards for validator ${block.generatorAddress} by ${commissionAmount}.`);
```

Comment:
I think this should be:
```suggestion
					logger.trace(`Increasing self-stake rewards for validator ${block.generatorAddress} by ${ selfStakeReward}.`);
					await validatorsTable.increment({
						increment: { totalSelfStakeRewards: BigInt(selfStakeReward) },
						where: { address: block.generatorAddress },
					}, dbTrx);
					logger.debug(`Increased self-stake rewards for validator ${block.generatorAddress} by ${ selfStakeReward}.`);
```